### DISCUSSION TITLE
How many branch can we create?

### DISCUSSION THREAD

**Discussion by SenseiRofu** [community/community#28335] [Other Feature Feedback, Questions, & Ideas] [answered]
Are there any limit about this.

**Comment by osmnnunl** [comment]
I think infinite :D

**Comment by 8mehmet11** [comment, marked as answer]
There is no hard limit on the number of branches, tags, remote-tracking names, and other references. (All of Git's name-to-hash-ID map entries are refs or references: branch names are just refs whose full name starts with refs/heads/ ). These are not always stored in
